app/controllers/api/v1/base_controller.rb

```diff
@@ -81,6 +81,6 @@ def api_error(status: 500, errors: [])
   private
 
     def paginated_url(base_url, page)
-      page ? "#{base_url}?page[number]=#{page}" : nil
+      "#{base_url}#{base_url.index('?').nil? ? '?' : '&'}page=#{page}" if page
```

Comment:
Why do we need to change this ?

Reply:
because base_url can have params like in the case of search, it has query params

Reply:
yup, there will be page[number], I forgot to make that changes locally, it was done on gitpod while testing.

Reply:
Could you expand this to multiple line using if statements ?